### ISSUE TITLE
Epochs

### ISSUE THREAD

**Issue by gustavo-stefano** [michhar/pytorch-yolo-v3-custom#15]
Is there a variable that I can change the number of epochs? I am able to train on my costume data. The train is very quick, less than a minute and it runs only 3 epochs. I am training on 200 images and tried to unfreeze different number of layers. I also tried to change the accuracy, but the result is horrible. I have attached
![roundabout49_out](https://user-images.githubusercontent.com/38964186/76773578-ff12b380-67a2-11ea-9eb4-1fbfeb80f144.jpg)
![roundabout49_out](https://user-images.githubusercontent.com/38964186/76773640-1782ce00-67a3-11ea-99d1-a6112609163e.jpg)
![roundabout75_out](https://user-images.githubusercontent.com/38964186/76773642-181b6480-67a3-11ea-98eb-8a907a699fb9.jpg)

 an example of what I am getting.
